client/src/components/MessageList.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react"
+import { Link } from "react-router-dom"
+
+const MessageList = (props) => {
+    const [messageList, setMessageList] = useState([])
+    const getMessages = async () => {
+        try {
+            const response = await fetch("/api/v1/messages")
+            if (!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw error
+            }
+            const responseBody = await response.json()
+            setMessageList(responseBody.messageList)
+        } catch (error) {
+            console.error(`Error in fetch: ${error.message}`)
+        }
+    }
+    useEffect(() => {
+        getMessages()
+    }, [])
+
+    const messagesToRender = messageList.map((message) => {
+        return (
+            <div key={message.id}>
+                id={message.id}
+                userId={message.userId}
+                content={message.content}
+                image={message.image}
+            </div>
+        )
+    })
+    return (
+        <div>
+            <h1>Welcome to Leland Stanford Community MessageBoard</h1>
+            <div>
+                <Link to ="/new-message" className="button">
+                    <h3>Add New Message</h3>
```

Comment:
if we don't have a page for this yet we shouldn't add placeholders or filler text - instead, when you are building the feature to Add a New Message, you can add this link at that time